### ISSUE TITLE
report.app and report.appState are opposite 

### ISSUE THREAD

**Issue by woshimaliang** [bugsnag/bugsnag-cocoa#658]
### Description

According to header doc(https://github.com/bugsnag/bugsnag-cocoa/blob/master/Source/BugsnagCrashReport.h#L201-L208):

```
/**
 *  App information such as the name, version, and bundle ID
 */
@property(readwrite, copy, nullable) NSDictionary *app;
/**
 *  Device state such as oreground status and run duration
 */
@property(readwrite, copy, nullable) NSDictionary *appState;

```

https://github.com/bugsnag/bugsnag-cocoa/blob/master/Source/BugsnagCrashReport.m#L279-L283

```
            _app = BSGParseApp(report);
            _appState = BSGParseAppState(report[BSGKeySystem],
                                         BSGLoadConfigValue(report, @"appVersion"),
                                         _releaseStage, // Already loaded from config
                                         BSGLoadConfigValue(report, @"codeBundleId"));
```

### Issue
https://github.com/bugsnag/bugsnag-cocoa/blob/master/Source/BugsnagKSCrashSysInfoParser.m#L68-L119

However BSGParseApp is really for **appState**

```
   NSDictionary *BSGParseApp(NSDictionary *report) {
    NSDictionary *system = report[BSGKeySystem];

    NSMutableDictionary *appState = [NSMutableDictionary dictionary];
    
    NSDictionary *stats = system[@"application_stats"];
    
    NSInteger activeTimeSinceLaunch =
    [stats[@"active_time_since_launch"] doubleValue] * 1000.0;
    NSInteger backgroundTimeSinceLaunch =
    [stats[@"background_time_since_launch"] doubleValue] * 1000.0;
    
    BSGDictSetSafeObject(appState, @(activeTimeSinceLaunch),
                         @"durationInForeground");

    BSGDictSetSafeObject(appState, system[BSGKeyExecutableName], BSGKeyName);
    BSGDictSetSafeObject(appState,
                         @(activeTimeSinceLaunch + backgroundTimeSinceLaunch),
                         @"duration");
    BSGDictSetSafeObject(appState, stats[@"application_in_foreground"],
                         @"inForeground");
    BSGDictSetSafeObject(appState, system[@"CFBundleIdentifier"], BSGKeyId);
    return appState;
}
```

while 

```
NSDictionary *BSGParseAppState(NSDictionary *report, NSString *preferredVersion, NSString *releaseStage, NSString *codeBundleId) {
    NSMutableDictionary *app = [NSMutableDictionary dictionary];

    NSString *version = preferredVersion ?: report[@"CFBundleShortVersionString"];

    BSGDictSetSafeObject(app, report[@"CFBundleVersion"], @"bundleVersion");
    BSGDictSetSafeObject(app, releaseStage,
                         BSGKeyReleaseStage);
    BSGDictSetSafeObject(app, version, BSGKeyVersion);
    
    BSGDictSetSafeObject(app, codeBundleId, @"codeBundleId");
    
    NSString *notifierType;
#if TARGET_OS_TV
    notifierType = @"tvOS";
#elif TARGET_IPHONE_SIMULATOR || TARGET_OS_IPHONE
    notifierType = @"iOS";
#elif TARGET_OS_MAC
    notifierType = @"macOS";
#endif
    
    if ([Bugsnag configuration].notifierType) {
        notifierType = [Bugsnag configuration].notifierType;
    }
    BSGDictSetSafeObject(app, notifierType, @"type");
    return app;
}
```

is for **app**?

### Environment

Library versions:

* Bugsnag version : master
* iOS/tvOS/macOS version(s): iOS
* debug mode or production?: Debug

**Comment by xander-jones** [comment]
Hey @woshimaliang, thanks for reporting this and raising a PR. A fix for this is included in our latest update for v6 of this notifier which is undergoing testing at the moment. It will be released soon!

**Comment by jparise** [comment]
See also #572.

**Comment by mattdyoung** [comment]
Fixed in [v6.0.0](https://github.com/bugsnag/bugsnag-cocoa/releases/tag/v6.0.0).

Note: There are a number of breaking changes in v6.0.0 as part of an effort to improve the interface and make it more consistent across our error reporting libraries, so please refer to the upgrade guide:
https://github.com/bugsnag/bugsnag-cocoa/blob/master/UPGRADING.md
